qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -74,13 +158,73 @@ def __init__(
             configuration: Backend configuration.
             api_client: IBM Quantum client used to communicate with the server.
         """
-        super().__init__(configuration=configuration)
-
+        super().__init__(
+            name=configuration.backend_name,
+            online_date=configuration.online_date,
+            backend_version=configuration.backend_version,
+        )
         self._api_client = api_client
-
-        # Attributes used by caching functions.
+        self._configuration = configuration
         self._properties = None
+        self._qubit_properties = None
         self._defaults = None
+        self._target = None
+        self._max_circuits = configuration.max_experiments
+        if not self._configuration.simulator:
+            self.options.set_validator("noise_model", type(None))
+            self.options.set_validator("seed_simulator", type(None))
+
+    def __getattr__(self, name: str) -> Any:
+        """Gets attribute from self or configuration
+
+        This magic method executes when user accesses an attribute that
+        does not yet exist on IBMBackend class.
+        """
+        # Lazy load properties and pulse defaults and construct the target object.
+        self._get_properties()
+        self._get_defaults()
```

Comment:
Did you look into lazy loading this via an `InstructionProperties` subclass like we discussed offline? I'm concerned about the performance overhead if we're required to fetch and parse the defaults payload for ever use of a backend with the qiskit compiler.

Reply:
Merging this as is for the rc2 release, I will handle this in a separate PR.